solver/src/solver/http_solver/settlement.rs

```diff
@@ -448,6 +449,20 @@ mod tests {
                         }
                     ]
                 },
+                "1": {
```

Comment:
was this test case failing without the one line change above? If not, can we please make it so?

Reply:
It certainly was. Otherwise I wouldn't have added this.